Assets/Scripts/Constraints/RestrictionIssueDeltaSet.cs

```diff
@@ -1,80 +1,82 @@
 ﻿using System.Collections.Generic;
 using Newtonsoft.Json.Linq;
-using Newtonsoft.Json;
 
-/// <summary>
-/// A delta set that specifies the changes in issues.
-/// </summary>
-public class RestrictionIssueDeltaSet
+namespace MSP2050.Scripts
 {
-	private HashSet<PlanIssueObject> addedIssues = new HashSet<PlanIssueObject>(IssueObjectEqualityComparer.Instance);
-	private List<PlanIssueObject> removedIssues = new List<PlanIssueObject>();
-
-	public void IssueRemoved(PlanIssueObject removed)
+	/// <summary>
+	/// A delta set that specifies the changes in issues.
+	/// </summary>
+	public class RestrictionIssueDeltaSet
 	{
-		if (!addedIssues.Remove(removed))
+		private HashSet<PlanIssueObject> addedIssues = new HashSet<PlanIssueObject>(IssueObjectEqualityComparer.Instance);
+		private List<PlanIssueObject> removedIssues = new List<PlanIssueObject>();
+
+		public void IssueRemoved(PlanIssueObject removed)
 		{
-			bool result = removedIssues.Contains(removed);
-			if (result == false)
-			{
-				removedIssues.Add(removed);
-			}
-			else
+			if (!addedIssues.Remove(removed))
 			{
-				UnityEngine.Debug.LogError("Issue is already in the removed issues delta set.");
+				bool result = removedIssues.Contains(removed);
+				if (result == false)
+				{
+					removedIssues.Add(removed);
+				}
+				else
+				{
+					UnityEngine.Debug.LogError("Issue is already in the removed issues delta set.");
+				}
 			}
 		}
-	}
 
-	public void IssueAdded(PlanIssueObject added)
-	{
-		if (!removedIssues.Remove(added))
+		public void IssueAdded(PlanIssueObject added)
 		{
-			bool result = addedIssues.Add(added);
-			if (!result)
+			if (!removedIssues.Remove(added))
 			{
-				UnityEngine.Debug.LogError("Issue already in the added delta set.");
+				bool result = addedIssues.Add(added);
+				if (!result)
+				{
+					UnityEngine.Debug.LogError("Issue already in the added delta set.");
+				}
 			}
 		}
-	}
 
-	public IEnumerable<PlanIssueObject> GetAddedIssues()
-	{
-		return addedIssues;
-	}
+		public IEnumerable<PlanIssueObject> GetAddedIssues()
+		{
+			return addedIssues;
+		}
 
-	public IEnumerable<PlanIssueObject> GetRemovedIssues()
-	{
-		return removedIssues;
-	}
+		public IEnumerable<PlanIssueObject> GetRemovedIssues()
+		{
+			return removedIssues;
+		}
 
-	public bool HasAnyChanges()
-	{
-		return addedIssues.Count > 0 || removedIssues.Count > 0;
-	}
+		public bool HasAnyChanges()
+		{
+			return addedIssues.Count > 0 || removedIssues.Count > 0;
+		}
 
-	public void SubmitToServer(BatchRequest batch)
-	{
-		if (HasAnyChanges())
+		public void SubmitToServer(BatchRequest batch)
 		{
-			//form.AddField("added", GetAddedIssues());
-			//form.AddField("removed", GetRemovedIssues());
+			if (HasAnyChanges())
+			{
+				//form.AddField("added", GetAddedIssues());
```

Comment:
Commented code, as always see if it makes sense to leave it there for the future, otherwise remove it! :)